src/components/interactive-lab-callout/interactive-lab-callout.module.css

```diff
@@ -11,11 +11,17 @@
 		display: flex;
 		gap: 32px;
 	}
+
+	@nest html[data-theme='dark'] & {
+		background-color: var(
+			--token-color-foreground-strong
+		); /** Needed to supercede the dark mode card .root dark mode background  */
+	}
 }
 
 .title {
 	composes: hds-typography-body-200 from global;
-	color: var(--token-color-surface-primary);
+	color: var(--token-color-surface-faint);
```

Comment:
From what I'm seeing in the Figma, the title should still be surface/primary (although the body text _is_ surface/faint):
https://www.figma.com/file/v0Hyqvjdf6QJAvRkdIBV24/Dark-Mode?node-id=598-35371&t=fVgpqBU7D48pQX2o-0

![CleanShot 2023-03-21 at 21 19 20](https://user-images.githubusercontent.com/43934258/226777336-ef7bf503-cabf-4de2-a529-a230f15f22b4.png)

Reply:
Oh good catch! thank you 🙇🏻